### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by romanWienicke**
Setup on Windows needs some addtional checks and settings or the install failes.

### CODE REVIEW

File: source/developers-guide/vagrant-phpstorm/index.md [hunk 1]
```diff
@@ -22,6 +22,12 @@ Download the required software:
 
  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
  - [Vagrant](https://www.vagrantup.com/downloads)
+  
+ Additional Information for VirtualBox setup (Windows), or you get errors like "error downloading the file" without any further exlanation:
+- make shure VirtualBox supports 64bit Guest Systems, if only 32 bit is supported check if:
+  - your hos OS is a 64 bit system
+  - Intel Virtualiuation Technology and VT-d are both(!) enabled
```
Comment (alfredbez) [inline source/developers-guide/vagrant-phpstorm/index.md]: Intel Virtuali**z**ation Technology

File: source/developers-guide/vagrant-phpstorm/index.md [hunk 2]
```diff
@@ -22,6 +22,12 @@ Download the required software:
 
  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
  - [Vagrant](https://www.vagrantup.com/downloads)
+  
+ Additional Information for VirtualBox setup (Windows), or you get errors like "error downloading the file" without any further exlanation:
+- make shure VirtualBox supports 64bit Guest Systems, if only 32 bit is supported check if:
+  - your hos OS is a 64 bit system
```
Comment (alfredbez) [inline source/developers-guide/vagrant-phpstorm/index.md]: hos -> hos**t**

File: source/developers-guide/vagrant-phpstorm/index.md [hunk 3]
```diff
@@ -22,6 +22,12 @@ Download the required software:
 
  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
  - [Vagrant](https://www.vagrantup.com/downloads)
+  
+ Additional Information for VirtualBox setup (Windows), or you get errors like "error downloading the file" without any further exlanation:
+- make shure VirtualBox supports 64bit Guest Systems, if only 32 bit is supported check if:
```
Comment (alfredbez) [inline source/developers-guide/vagrant-phpstorm/index.md]: shure -> sure

File: source/developers-guide/vagrant-phpstorm/index.md [hunk 4]
```diff
@@ -22,6 +22,12 @@ Download the required software:
 
  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
  - [Vagrant](https://www.vagrantup.com/downloads)
+  
+ Additional Information for VirtualBox setup (Windows), or you get errors like "error downloading the file" without any further exlanation:
```
Comment (alfredbez) [inline source/developers-guide/vagrant-phpstorm/index.md]: exlanation -> ex**p**lanation

**Comment by mitelg** [comment]
Thanks @alfredbez for your code review!
@romanWienicke it would be nice, if you could change these things :+1:

**Pull request merged**